### PR TITLE
feat: Alert Component added in documentation

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added alert component in documentation.

Live: https://deploy-preview-7--ui-component-library123213.netlify.app/documentation

### CODE REVIEW

File: Documentation/Docs-CSS/main-section.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+.component-text {
+  color: var(--white);
+}
+
+.component-display {
+  margin: 2rem 0;
+  padding: 2rem 1rem;
+  background-color: var(--secondary);
+  /* box-shadow: inset 6px 6px 12px #131313, inset -6px -6px 12px #292929; */
```
Comment (Maianki) [inline Documentation/Docs-CSS/main-section.css]: remove code that is unused.

File: Documentation/Docs-CSS/main-section.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+.component-section {
+  width: 90vw;
+  padding: 1rem;
+  margin: 1rem auto;
+  height: fit-content;
+  background-color: var(--secondary);
+  color: var(--primary);
+  border-radius: 20px;
+  box-shadow: 6px 6px 12px #131313, -6px -6px 12px #292929;
+}
```
Comment (Maianki) [inline Documentation/Docs-CSS/main-section.css]: Try using CSS variable here to be consistent.

**Review by Maianki — CHANGES_REQUESTED** [review]
I have suggested some minor changes. Everything else looks good.

File: Documentation/Docs-CSS/main-section.css (hunk 1 above)
Comment (ParthParolekar) [reply Documentation/Docs-CSS/main-section.css]: > I have suggested some minor changes. Everything else looks good.

Yup! I Will do it. Thanks for the review!

**Review by Maianki — APPROVED** [review]
Looks good you can merge

**Comment by yash9263** [comment]
looks good to me

**Pull request merged**